test/applyMiddleware.spec.ts

```diff
@@ -51,16 +51,11 @@ describe('applyMiddleware', () => {
     const spy = jest.fn()
     const store = applyMiddleware(test(spy), thunk)(createStore)(reducers.todos)
```

Comment:
This `store` was not being typed properly, because the `thunk` wasn't typed properly. That's now fixed in this PR.